src/utils/iconInterface.ts

```diff
@@ -0,0 +1,17 @@
+export interface SuomifiIconProps {
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Aria-label for SVG, undefined hides SVG from screen reader
+   * @default undefined
+   */
+  ariaLabel?: string;
+  /** Show mouse cursor as hand-pointer */
+  mousePointer?: boolean;
+  /** Custom fill color */
+  color?: string;
+  /** Custom fill color, takes precedence over color if both are provided */
+  fill?: string;
+  testId?: string;
```

Comment:
What is this used for?

Reply:
This is from interface that was in suomifi-ui-components Icon.tsx so keeping it the same for now.